mantle/kola/tests/ignition/luks.go

```diff
@@ -50,6 +50,17 @@ func init() {
 		ExcludeArchitectures: []string{"s390x"}, // no TPM backend support for s390x
 		Tags:                 []string{"luks", "tpm", "tang", "sss", kola.NeedsInternetTag, "reprovision"},
 	})
+	register.RegisterTest(&register.Test{
```

Comment:
I think the way this test is intended to be run is with `kola run luks.cex --qemu-cex` but doing separate kola invocations adds complexity in the pipeline. So if we can avoid it, we should just run it as part of the regular kola runs.

(We didn't do it this way for Secure Execution because it requires a separate artifact which is built later in the pipeline. But here, we can just run this test directly on the built QEMU image.)

To do this, you can add a new `Cex` knob to the `platform.QemuMachineOptions` struct. Then you would set that knob to `true` when creating the QEMU VM in your test here (using `NewMachineWithQemuOptions` instead of `NewMachineWithOptions`). See `runBasicTests` for an example of this.

Reply:
Hi @jlebon ,
If i understand correctly the calling of the test will be basic run without providing `luks.cex` similar to the test mentioned in the `runBasicTests`? The codes will be remain same at `luks.go` ?